.mailmap

```diff
@@ -244,3 +244,7 @@ Abhinav Agarwal <abhinavagarwal1996@gmail.com> abhinavagarwal07 <abhinavagarwal1
 Rishabh Daal <rishabhdaal@gmail.com> rd13123013 <rishabhdaal@gmail.com>
 Aravind Reddy <aravindreddy255@gmail.com> Aravind <aravindreddy255@gmail.com>
 Akshay Siramdas <akshaysiramdas@gmail.com> AkshaySiramdas <akshaysiramdas@gmail.com>
+Nitin Chaudhary <nitinmax1000@gmail.com> Nitin <nitinmax1000@gmail.com>
+Alex Argunov <sajkoooo@gmail.com> alex argunov <sajkoooo@gmail.com>
+Abhishek Garg <abhishekgarg119@gmail.com> abhishek garg <abhishekgarg119@gmail.com>
+Gaurav Dhingra <igauravdhingra@protonmail.com> Gaurav Dhingra <axyd0000@gmail.com>
```

Comment:
Sorry. I assumed you preferred the other one because it was in newer commits, and is the one listed on your GitHub profile. I have pushed a commit fixing this.